### DISCUSSION TITLE
version == 1.6

### DISCUSSION THREAD

**Discussion by ttlns** [kaliiiiiiiiii/Selenium-Driverless#75] [Announcements]
### relevant changes
- change how iframes work (see Readme.md for an example, more reliable)
- add `driver.quit(clean_dirs=True)` argument
- make `driver.execute_script` use "deep" deserialization with Remote-Objects

### should fix
- https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/70
- https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/68
- https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/59
- https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/58
- https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/54

### bugfixes and minor changes
- change how the driver is quit (more reliable)
- improve stale element & RemoteObjects detection

**Comment by Fragaile** [comment]
Love the dedication to this project, awesome developer! ❤️

**Reply by ttlns** [reply]
Thanks!

**Reply by wealthr** [reply]
NICE, please when will you add finding element By.LINK_TEXT attribute

**Reply by ttlns** [reply]
I think this should be possible using XPATH?

**Reply by wealthr** [reply]
the sites element is dynamic, XPATH isn't a solution.

By.LINK_TEXT works with selenium profile without driverless, but not with driverless

**Reply by ttlns** [reply]
What's about `//a[text()='some_text']` ? If that doesn't work, use javascript.
